components/gitpod-db/src/typeorm/auth-provider-entry-db-impl.ts

```diff
@@ -45,11 +45,25 @@ export class AuthProviderEntryDBImpl implements AuthProviderEntryDB {
         await repo.update({ id }, { deleted: true });
     }
 
-    async findAll(): Promise<AuthProviderEntry[]> {
+    async findAll(exceptOAuthRevisions: string[] = []): Promise<AuthProviderEntry[]> {
+        const repo = await this.getAuthProviderRepo();
+        let query = repo.createQueryBuilder('auth_provider')
+            .where('auth_provider.deleted != true')
+        if (exceptOAuthRevisions.length > 0) {
+            query = query.andWhere('auth_provider.oauthRevision NOT IN (:exceptOAuthRevisions)', { exceptOAuthRevisions });
```

Comment:
this is new since last typeorm bump. 👏🏻  need to rewrite a handful of workarounds we've used before that. 

please check, that this actually works, according to the docs it's missing `...`, see https://github.com/typeorm/typeorm/blob/master/docs/select-query-builder.md#using-parameters-to-escape-data